app/Models/Product.php

```diff
@@ -39,4 +39,9 @@ public function images(){
 
         return $this->hasMany(Image::class);
     }
+
+    public function newPrice(){
+
+        return $this->price*(1 - $this->discount * 0.1);
```

Comment:
tại sao discount*0.1 em? lúc

Reply:
Dạ do e cho select số nguyên 1-10 a

Reply:
lý do em *0.1 ấy em

Reply:
em *0,1 để tính tỉ lệ giảm giá a, vì discount lúc đầu e selectRange số nguyên

Reply:
vậy anh giảm giá 20%, 50% thì làm như nào?

Reply:
20% thì tương ứng $this->discount = 2 , 50% thì $this->discount = 5. Rồi $this->discount * 0.1 tỉ lệ giảm giá a

Reply:
a vẫn chưa hiểu tỉ lệ giảm giá là gì cả. Em giải thích rõ hơn được không?